app/code/Magento/Captcha/Observer/CheckUserForgotPasswordBackendObserver.php

```diff
@@ -3,9 +3,18 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
 namespace Magento\Captcha\Observer;
 
+use Magento\Captcha\Helper\Data as CaptchaHelper;
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\ActionFlag;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\App\RequestInterface;
+use Magento\Framework\Event\Observer as Event;
```

Comment:
Would be better to name it as EventObserver

Reply:
I dissagre, it is not an observer. It is a container that is passed to an observer. The name `observer` is wrong for this type.

Reply:
Sry, didn’t saw in the details, just thought about full class name and alias, it’s not logical right now, could check it later

Reply:
Generally speaking, you have 2 "Observer" classes in Magento :man_facepalming: 
One is the published event with its data, which you can see above.
The second one is real Observer which is class listening for Event changes.